steps/core/spec.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
These are part of the runtime for steps in general and aren't specific to Relay Core. The only valid schema for this image could be the empty schema.

Reply:
Ah gotcha. Pushed an update. I'm mostly trying to squelch the error:

```
*impl.Error: schema_lookup_error: failed to lookup schema
github_com_puppetlabs_relay_core_pkg_workflow_validation_SchemaDoesNotExistError: schema for relaysh/core does not exist in the schema registry
```